lua/nvim-tree.lua

```diff
@@ -381,7 +383,9 @@ local function setup_autocommands(opts)
         (filters.config.filter_no_buffer or renderer.config.highlight_opened_files ~= "none")
         and vim.bo[data.buf].buftype == ""
       then
-        reloaders.reload_explorer(nil, data.buf)
+        utils.debounce("Buf:filter_buffer", opts.view.debounce_delay, function()
```

Comment:
Was the bug caused by having separate categories?

Reply:
It's simply because `view.debounce_delay` was used instead of `opt.view.debounce_delay`.